### ISSUE TITLE
FS Model & FT Model

### ISSUE THREAD

**Issue by BIGWangYuDong** [sshan-zhao/GASDA#11]
In Ft model, 
            [self.src_real_depth = input['src']['depth'].to(self.device)](url)
and in FS model,
            [self.fake_tgt = self.netG_Src(self.src_img).detach()](url)
I thought the ft model and the fs model is Contrary.

**Comment by sshan-zhao** [comment]
> In Ft model,
> [self.src_real_depth = input['src']['depth'].to(self.device)](url)
> and in FS model,
> [self.fake_tgt = self.netG_Src(self.src_img).detach()](url)
> I thought the ft model and the fs model is Contrary.

Yes, I just wrote like that at the beginning.
